client/components/telecom/telephony/group/number/feature/ovhPabx/menu/entry/edit/telephony-group-number-feature-ovh-pabx-menu-entry-edit.html

```diff
@@ -300,23 +300,25 @@ <h2 data-translate="telephony_number_feature_ovh_pabx_menu_entry_edit_title"></h
             <section data-ng-switch-when="dtmf"
                      class="dtmf-section">
                 <div class="section-group">
-                    <label class="section-item item-choice full-width pointer dtable">
-                        <div class="dtr">
-                            <div class="dtd vertical-middle radio-content">
-                                <flat-checkbox>
-                                    <input type="checkbox"
-                                           name="menuEntryAction"
-                                           data-ng-model="$ctrl.model.singleDigitKey"
-                                           data-ng-change="$ctrl.menuEntryCtrl.menuEntry.dtmf = ''" />
-                                </flat-checkbox>
-                            </div>
-                            <div class="dtd vertical-middle">
-                                <span class="block"
-                                      data-translate="telephony_number_feature_ovh_pabx_menu_entry_edit_dtmf_single_digit">
+                    <div class="oui-checkbox">
+                        <div class="section-item item-choice w-100 dtable">
```

Comment:
`dtable` => `d-table` (I'm not sure if it work well with `oui-checkbox`, if not we can get rid of this class).